agent/src/main/java/org/openremote/agent/protocol/tradfri/device/event/EventHandler.java

```diff
@@ -0,0 +1,34 @@
+package org.openremote.agent.protocol.tradfri.device.event;
+
+import java.lang.reflect.ParameterizedType;
+
+/**
+ * The class that handles events for IKEA TRÅDFRI devices
+ * @author Stijn Groenen
+ * @version 1.0.0
+ */
+public abstract class EventHandler<T extends Event> {
```

Comment:
The generic use of `Event` offers nothing for the reason mentioned above regarding `Event` being empty